docs/_source/practical_guides/create_dataset.md

```diff
@@ -572,50 +616,15 @@ for record in dataset.records:
 dataset.update_records(modified_records)
 ```
 
-###### Update `suggestions`
-
-You can also add suggestions to existing records that have been already pushed to Argilla:
-
-```python
-import argilla as rg
-
-rg.init(api_url="<ARGILLA_API_URL>", api_key="<ARGILLA_API_KEY>")
-
-dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
-```
-
-::::{tab-set}
-
-:::{tab-item} Argilla 1.14.0 or higher
-
-```python
-for record in dataset.records:
-    record.update(suggestions=[{"question_name": "question", "value": ...}]) # Directly pushes the update to Argilla
+```{note}
```

Comment:
@nataliaElv I think it is better to remove this indeed to avoid confusion

Reply:
Can you be more explicit about your concerns here? Maybe we can rephrase somehow?

Reply:
It is fine like this